app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -150,6 +151,17 @@ object TelemetryWrapper {
         val AVERAGE = "average"
     }
 
+
+    enum class BrowserContextMenuValue {
+        Link, Image, ImageWithLink;
+
+        override fun toString(): String = when(this) {
+            Link -> Value.LINK
+            Image -> Value.IMAGE
+            ImageWithLink -> Value.IMAGE_WITH_LINK
+       }
```

Comment:
\[ktlint\] WARNING: \[indent\] Unexpected indentation \(7\) \(it should be multiple of 4\) in column 1